src/index.html

```diff
@@ -2,11 +2,273 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="\images\logo.png"
+          alt="logo"
+          class="logo_img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="is-active">apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link"
+              data-qa="nav-hover">laptops & computers</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">gadgets</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">tablets</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">PHOTO</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+    <main>
+      <div class="ProductCard" data-qa="card">
+        <img class="ProductCard__Image"
+        src="\images\imac.jpeg" alt="ProductImage">
```

Comment:
Fix all cases if present
```suggestion
        <img 
          class="ProductCard__Image"
          src="\images\imac.jpeg" 
          alt="ProductImage"
        >
```